apps/hubble-stats/constants/index.ts

```diff
@@ -1,9 +1,46 @@
 import vAnchorClient from '@webb-tools/vanchor-client';
+import { getDateFromEpoch, getEpochFromDate } from '../utils';
 
-export const vAnchorAddresses = ['0x4b88368Eb14D7d09f0ca737832cEBfb8F12e3f05'];
+export const VANCHOR_ADDRESSES = ['0x7aA556dD0AF8bed063444E14A6A9af46C9266973'];
```

Comment:
Parameterize this as an environment variable and parse it in this context (use the current value as the default value):

```
# Set the VANCHOR_ADDRESSES as a JSON string array 
# export VANCHOR_ADDRESSES="[\"0x123\",\"0x234\"]"
VANCHOR_ADDRESSES=JSON.parse(process.env.VANCHOR_ADDRESSES)
```